src/default.nix

```diff
@@ -35,17 +35,19 @@
   # load from default directory
   else ./external,
 } @ args: let
+  argsConfig = config;
```

Comment:
I made this change because before if a `config` was not passed, and the config was loaded from the environment variable, the `loadConfig` below would not use that config. This is because `args.config != config` if the `config` was not passed and uses a default, `args.config` will simply just not exist.